strawberryfields/ops.py

```diff
@@ -1816,8 +1815,8 @@ class BipartiteGraphEmbed(Decomposition):
     rotation gates.
 
     Args:
-        A (array): An :math:`N\times N` complex or real symmetric matrix representing
-            a bipartite graph, where the number of vertices in the two vertex sets are even.
+        A (array): an :math:`N\times N` complex or real symmetric adjacency matrix representing
```

Comment:
It would be a good idea to have an option to pass either B = [[0,A],[A^T,0]] or A directly and infer which by how many qmodes are available....

Reply:
Yep I will make this change now

Reply:
Okay, I've added a keyword argument